update docs (#584)
replicatedhq/kurl.sh: src/release-notes/v2021.10.01-0.md

- Containerd is now the default container runtime, replacing the previous default container runtime, Docker.
- Log rotation will now be configured by default for the [Docker add-on](https://kurl.sh/docs/add-ons/docker), where the [max-size](https://docs.docker.com/config/containers/logging/json-file/#options) parameter for the log file is set to `10m`.
- Added the ability to configure log rotation through kubelet, which helps when using containerd instead of docker.

> - Re-Enabled Custom host preflight checks which can be declared in the kURL installer spec.

Reviewer: ```suggestion
- Re-enabled the ability to declare custom host preflight checks in the kURL installer spec.
```
